lib/service/state.go

```diff
@@ -27,23 +27,25 @@ import (
 
 const (
 	// stateOK means Teleport is operating normally.
-	stateOK = 0
-
+	stateOK = iota
```

Comment:
I'd prefer to reserve `iota` for use with internal constants only.  Because these values are exposed via the Prometheus gauge, it is better to explicitly define them (harder to accidentally make a breaking change via insertion or inconvenient merge).